resources/templates/ansic_cmake_connectionstring/CMakeLists-Windows-SourceCode.txt

```diff
@@ -0,0 +1,46 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#this is CMakeLists.txt for {PROJECT_NAME}
```

Comment:
Not quite sure why adding '-Windows-SourceCode' after CMakeLists. Is this a valid cmake file?

Reply:
It is just a template  CMakeList file name, which is used to identified for different platform and different sdk refernce type, it will use the standard "CMakeLists.txt" file name when persistent on disc.

---
In reply to: [336844826](https://github.com/microsoft/vscode-iot-workbench/pull/781#discussion_r336844826) [](ancestors = 336844826)